api/managementpb/actions.proto

```diff
@@ -282,6 +283,20 @@ message CancelActionRequest {
   ];
 }
 
+message StartPTSummaryActionRequest {
```

Comment:
Why we put it between `CancelActionRequest` and `CancelActionResponse`?

Reply:
I didnt notice that. Thanks.